api/src/wfl/service/slack.clj

```diff
@@ -94,9 +94,7 @@
   [{:keys [watchers uuid project labels] :as workload} message]
   (let [channels (filter slack-channel-watcher? watchers)
         project  (or project (util/label-value labels "project"))
-        swagger  (str/join "/" [(env/getenv "WFL_WFL_URL") "swagger"])
-        header   (format "%s workload %s *%s*"
-                         (link swagger "WFL") project uuid)]
+        header   (format "WFL workload %s *%s*" project uuid)]
```

Comment:
Same here: https://github.com/broadinstitute/wfl/pull/589/files#r823115500

A Slack message previously emitted from WFL prod but WFL link points to WFL dev: https://broadinstitute.slack.com/archives/C02P8TYQL21/p1646856557068449